apps/studio/src/types/schema.ts

```diff
@@ -0,0 +1,8 @@
+import { type JsonSchema7 } from '@jsonforms/core'
+
+export interface IsomerJsonSchema extends JsonSchema7 {
+  groups?: Array<{
```

Comment:
Yup, we are. The alternative is to come up with a separate UI schema which I want to avoid because it is a lot harder to maintain two schemas at the same time.